src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const Calculator = {
+    result: 0,
+
+    add(a, b) {
```

Comment:
it is better to give a more understandable name for the parameters

Reply:
I think that is enough for this case and not important because we can read this too easy.
But we always can improve our code - that's why I changed it, thanks.